seerep-hdf5/seerep-hdf5-core/include/seerep-hdf5-core/hdf5-core-tf.h

```diff
@@ -41,6 +41,9 @@ class Hdf5CoreTf : public Hdf5CoreGeneral
   inline static const std::string HDF5_GROUP_TF = "tf";
 
   inline static const std::string SIZE = "size";
+
+  inline static const std::string PARENT_FRAME = "PARENT_FRAME";
+  inline static const std::string CHILD_FRAME = "CHILD_FRAME";
```

Comment:
if you introduce these, also use them in `hdf5-core-tf.cpp` please

Reply:
I now changed all sections with hard coded "PARENT_FRAME" and "CHILD_FRAME" strings to use these constants.